vimish-fold.el

```diff
@@ -415,6 +415,74 @@ This feature needs `avy' package."
            folds-before-point)))
       (message "No more folds before point"))))
 
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Folding marks
+
+(defcustom vimish-fold-marks '("{{{" . "}}}")
```

Comment:
I think it should be just plain `nil`, not `'(nil . nil)`, there is no reason to use a more complex construction when the feature is disabled. The `nil` value should also be documented in the doc string. You'll also want to update `:type`.

Reply:
done